### DISCUSSION TITLE
Example of setting a custom TTL for RedisActorSessionStore?

### DISCUSSION THREAD

**Discussion by miketwenty1** [actix/actix-extras#401] [Q&A] [answered]
Does anyone have an example of setting a custom TTL for a actix middleware redis session? The default is 24 hours, I would like to set some of these sessions to a different Duration. 

Also, calling `renew` on a session, is it possible to renew with a custom TTL?

**Comment by robjtede** [comment, marked as answer]
Copied from Discord:

sure, this applies to all backends: https://docs.rs/actix-session/latest/actix_session/config/struct.PersistentSession.html#method.session_ttl 

> calling renew on a session, is it possible to renew with a custom TTL?

This is not possible with the current lib.
